test/test_array_ops.cpp

```diff
@@ -316,6 +316,55 @@ TEST(ArrayOps, ExpandDims) {
 
 }  // end of test op ExpandDims
 
+// Test op:GatherND
+TEST(ArrayOps, GatherNd2D) {
+  int dim1 = 2;
+  int dim2 = 2;
+
+  Tensor indices(DT_INT32, TensorShape({2, 1}));
+  AssignInputValues<int>(indices, {0, 1});
+
+  Tensor params(DT_FLOAT, TensorShape({dim1, dim2}));
+  AssignInputValuesRandom(params);
+
+  vector<int> static_input_indexes = {};
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  Scope root = Scope::NewRootScope();
+  auto R = ops::GatherNd(root, params, indices);
+  std::vector<Output> sess_run_fetchoutputs = {R};
+
+  OpExecuter opexecuter(root, "GatherNd", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+
+}  // end of test op GatherND
+
+// Test op:GatherND
+TEST(ArrayOps, GatherNd3D) {
+  Tensor indices(DT_INT32, TensorShape({2, 1, 1}));
+  AssignInputValues<int>(indices, {0, 1});
+
+  Tensor params(DT_FLOAT, TensorShape({2, 2, 2}));
+  AssignInputValuesRandom(params);
+
+  vector<int> static_input_indexes = {};
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  Scope root = Scope::NewRootScope();
+  auto R = ops::GatherNd(root, params, indices);
+  std::vector<Output> sess_run_fetchoutputs = {R};
+
+  OpExecuter opexecuter(root, "GatherNd", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+
+}  // end of test op GatherND
+
 // Test op: Gather. vector indices
```

Comment:
These C++ tests will fail on GPU as GatherNd is not supported on GPU. `buildkite/ngtf-r0-dot-13-cpu-ubuntu-18-04`

Reply:
To pass CI on GPU, please add this test to the filter here: https://github.com/tensorflow/ngraph-bridge/blob/master/test/ci/buildkite/test_runner.py#L97